homework-16/custom.checker.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  calculatorPlus,
+  calculatorMinus,
+  calculatorDivision,
+  calculatorMultication,
+} from "./calculator";
+
+export function customChecker(
+  firstNumber: number,
+  secondNumber: number,
+  result: number,
+  sign: string
```

Comment:
Please be more specific in the `sign` variable possible values using TypeScript syntax.